.github/workflows/tests.yml

```diff
@@ -0,0 +1,41 @@
+name: brew test-bot
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  test-bot:
+    strategy:
+      matrix:
+        os: [ubuntu-22.04, macos-13, macos-14]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Cache Homebrew Bundler RubyGems
+        uses: actions/cache@v4
+        with:
+          path: ${{ steps.set-up-homebrew.outputs.gems-path }}
+          key: ${{ matrix.os }}-rubygems-${{ steps.set-up-homebrew.outputs.gems-hash }}
+          restore-keys: ${{ matrix.os }}-rubygems-
```

Comment:
Is there supposed to be another value at the end? `${{ matrix.os }}-rubygems-?`

Reply:
I don't believe so.  From the action documentation : restore-keys - An ordered multiline string listing the prefix-matched keys, that are used for restoring stale cache if no cache hit occurred for key.

This is how it looks like in this action:
```
    key: ubuntu-22.04-rubygems-ac6a885522f09004b07b6e43822b683037847dda494cf58d8756c200e74247bd
    restore-keys: ubuntu-22.04-rubygems-
```

It will match since it is prefix-matched to the cache key. 

I think this is intended to support multiple cache keys, whereas we only use one in this action.